File: Converter-Temperatura/asset/js/ConverterService.js
var ConverterService =
{
    getIndex: function (celsius, callback) {
        $.ajax({
            url: 'service/converter.aspx.cs',
            data: { 'celsius': celsius },
            success: function (result) {
                callback(parseFloat(result));
            },
            error: function () {
                callback(null);
            }
        });
    }
};

File: Converter-Temperatura/service/ConverterService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Converter_Temperatura.service
{
    public class ConverterService
    {
        public static double GetIndex(double celsius) 
        {
            return (celsius * 1.8) + 32;
        }
    }
}